docs/api/users-update-by-id.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+---
+
+# Update user by ID
+
+Allows the editing of [`user`](user) objects that already exist. You will need to know the `id` parameter of the user you want to edit.
+
+> [!IMPORTANT]
+> The information for the user is replaced **entirely** by the request body, so if you want to keep information the same you still need to include it in the request body.
+
+## URL
+
+```shell
+{server_url}/users/{id}
+```
+
```

Comment:
What HTTP method do you use for this call?

Reply:
Added the method!